ui/templates/base.html

```diff
@@ -11,6 +11,18 @@
     <title>{% block title %}{% endblock %}</title>
     {% block extrahead %}
     {% endblock %}
+    <!-- Google Analytics -->
+    <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+    {% load ui_extras %}
+    ga('create', '{% settings_value "GA_TRACKING_ID" %}', 'auto');
```

Comment:
Can you pass in `{{ GA_TRACKING_ID }}` instead and set the value in the context instead? That would avoid the need for the `templatetags` directory